modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/DartDioNextClientCodegen.java

```diff
@@ -333,7 +333,8 @@ private void appendBuiltValueCollection(StringBuilder sb, CodegenProperty proper
                 }
             }
 
-            if (op.allParams.stream().noneMatch(param -> param.dataType.equals("Uint8List"))) {
+            if (op.allParams.stream().noneMatch(param -> param.dataType.equals("Uint8List")) && op.responses.stream()
+                    .noneMatch(response -> response.dataType != null && response.dataType.equals("Uint8List"))) {
```

Comment:
`response -> response.dataType != null` can you put this condition in `filter()`?